extensions/bulk/index.md

```diff
@@ -91,8 +91,8 @@ Accept: application/vnd.api+json; ext=bulk
 Multiple resources can be deleted by sending a `DELETE` request to a URL that
 represents a collection of resources to which they all belong.
 
-The body of the request **MUST** contain a `data` member whose value is an
-an array of objects that each contain a `type` and `id`.
+The body of the request **MUST** contain a `"data"` member whose value is an
+an array of resource identifier objects.
```

Comment:
I would link this to the resource identifier objects section (same for all references).